transforms/ember-object/__testfixtures__/options/decorators-disabled.input.js

```diff
@@ -4,74 +4,74 @@ import { inject as service } from "@ember/service";
 import { on } from "@ember/object/evented";
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo1 = EmberObject.extend({
   computed: computed({
     get() {},
     set() {}
   })
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo2 = EmberObject.extend({
   statefulObject: {},
   statefulArray: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo3 = EmberObject.extend({
   tagName: "div"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo4 = EmberObject.extend({
   classNames: []
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo5 = EmberObject.extend({
   classNameBindings: ["foo"],
   foo: "val"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo6 = EmberObject.extend({
   attributeBindings: ["foo"],
   foo: "val"
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo7 = EmberObject.extend({
```

Comment:
**[Not a blocker]** This really seems transformable, and given its popularity, it's probably not great that we don't try.

I think this would mostly "just work", but even if there are caveats, I think it would be good to
have an option to do it anyway.

```ts
class Foo7 extends EmberObject {
  @action bar() {}
}
```

Reply:
We do transform this as long as decorators aren't disabled (as they are in this test).